src/nvim/tui/tui.c

```diff
@@ -53,16 +53,16 @@
 
 #define TOO_MANY_EVENTS 1000000
 #define STARTS_WITH(str, prefix) (strlen(str) >= (sizeof(prefix) - 1) \
-    && 0 == memcmp((str), (prefix), sizeof(prefix) - 1))
-#define TMUX_WRAP(is_tmux, seq) ((is_tmux) \
-    ? "\x1bPtmux;\x1b" seq "\x1b\\" : seq)
+                    && 0 == memcmp((str), (prefix), sizeof(prefix) - 1))
```

Comment:
Should have `(strlen` to this line because moving `&&` in this column doesn't make sense. `src/clint.py` does not work well with macros.

Reply:
I kept the macro definition in two lines because otherwise it would be too long.